entrypoint.sh

```diff
@@ -59,7 +59,7 @@ push_resource() {
     #   apis/*
     #   api.yaml
     # This specificity ensures that we avoid pushing when only the components subdir has changes.
-    if echo "$changed_files" | grep -qP "^${escaped_location}/((application|page|api).yaml|apis/)" ; then
+    if echo "$changed_files" | grep -qP "${escaped_location}/((application|page|api).yaml|apis/)" ; then
```

Comment:
Would it be useful to print `changed_files`?

Reply:
not really. regardless, any debugging enhancements should go in a different PR. lets prioritize the fix here